packages/0x.js/package.json

```diff
@@ -17,17 +17,15 @@
     "scripts": {
         "watch": "tsc -w",
         "prebuild": "run-s clean generate_contract_wrappers",
-        "build": "run-p build:umd:prod build:commonjs; exit 0;",
+        "build": "tsc && copyfiles -u 2 './src/compact_artifacts/**/*.json' ./lib/src/compact_artifacts && copyfiles -u 3 './lib/src/monorepo_scripts/**/*' ./scripts",
         "generate_contract_wrappers": "abi-gen --abis 'src/compact_artifacts/@(Exchange|Token|TokenTransferProxy|EtherToken|TokenRegistry|DummyToken).json' --template ../contract_templates/contract.handlebars --partials '../contract_templates/partials/**/*.handlebars' --output src/contract_wrappers/generated --backend ethers && prettier --write 'src/contract_wrappers/generated/**.ts'",
         "lint": "tslint --project .",
         "test:circleci": "run-s test:coverage",
-        "test": "run-s clean test:commonjs",
+        "test": "yarn run_mocha",
+        "rebuild-and-test": "run-s build test",
```

Comment:
@albrow moving forward let's try and keep `yarn script` names consistent by always using underscores over dashes. I've gone ahead and fixed these in a PR I will open soon.